Description: Lower expire timestamp in test.
 If this number is higher than signed 32b int, tests are failing on 32
 bit systems. This will brake tests of Swift3 in 2038, sorry for it :).
Author: Ondřej Nový <novy@ondrej.org>
Forwarded: https://review.openstack.org/#/c/283274/
Last-Update: 2016-02-22

--- swift-plugin-s3-1.10.orig/swift3/test/unit/test_middleware.py
+++ swift-plugin-s3-1.10/swift3/test/unit/test_middleware.py
@@ -166,7 +166,9 @@ class TestSwift3Middleware(Swift3TestCas
         self.assertEquals(self._get_error_code(body), 'AccessDenied')
 
     def test_signed_urls(self):
-        expire = '10000000000'
+        # Set expire to last 32b timestamp value
+        # This number can't be higher, because it breaks tests on 32b systems
+        expire = '2147483647' # 19 Jan 2038 03:14:07
         req = Request.blank('/bucket/object?Signature=X&Expires=%s&'
                             'AWSAccessKeyId=test:tester' % expire,
                             environ={'REQUEST_METHOD': 'GET'},
